modules/distr-tracing-otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -6,7 +6,35 @@ This module is included in the following assemblies:
 [id="distr-tracing-send-metrics-monitoring-stack_{context}"]
 = Sending metrics to the monitoring stack
 
-You can configure the monitoring stack to scrape OpenTelemetry Collector metrics endpoints and to remove duplicated labels that the monitoring stack has added during scraping.
+The OpenTelemetry Collector custom resource can be configured to create Prometheus `ServiceMonitor` to scrape collector's pipeline metrics and enabled Prometheus exporters.
```

Comment:
Question: the `ServiceMonitor` is what? a CR or a what exactly? Let's define it here (to avoid getting asked to do this by the merge reviewer).
```suggestion
You can configure the OpenTelemetry Collector custom resource to create a Prometheus `ServiceMonitor` to scrape the collector's pipeline metrics and enabled Prometheus exporters.
```